src/plugins/mer/meremulatordevice.h

```diff
@@ -168,6 +176,9 @@ private slots:
     static MerEmulatorDeviceManager *s_instance;
     QHash<Core::Id, quint16> m_deviceSshPortCache;
     QHash<Core::Id, Utils::PortList> m_deviceQmlLivePortsCache;
+    QSet<QString> m_editedDeviceModels;
+
+    static void onDeviceModelsChanged(const QSet<QString> &deviceModelNames);
```

Comment:
I would prefer having member variables declared at the very end of class definition.

Reply:
Fixed.